tests/cluster/eh/test_eh_saving.py

```diff
@@ -1,14 +1,23 @@
+import time
 import unittest
+from datetime import datetime, timedelta
 
 from pyspark.sql import functions as f
 
+from atc.config_master import TableConfigurator
+from atc.eh import EventHubCapture
+from atc.functions import init_dbutils
 from atc.spark import Spark
 
 from .AtcEh import AtcEh
 
 
 class EventHubsTests(unittest.TestCase):
-    def test_publish(self):
+    @classmethod
+    def setUpClass(cls) -> None:
+        TableConfigurator().clear_all_configurations()
```

Comment:
This is not in debug mode - is that on purpose?

Reply:
It makes no difference here since the registered table contains no tags like `{ID}`, so debug and release table are the same.